semgrep/semgrep/test.py

```diff
@@ -195,62 +198,75 @@ def confusion_matrix_to_string(confusion: List[int]) -> str:
     return f"TP: {tp}\tTN: {tn}\tFP: {fp}\tFN: {fn}"
 
 
+def invoke_semgrep_multi(filename, *args, **kwargs):
+    try:
+        output = invoke_semgrep(filename, *args, **kwargs)
+    except Exception as error:
+        return (filename, error, {})
+    else:
+        return (filename, None, output)
```

Comment:
```suggestion
    try:
        output = invoke_semgrep(filename, *args, **kwargs)
        return (filename, None, output)
    except Exception as error:
        return (filename, error, {})
```

Reply:
I try to only include the line I'm expecting an exception from in the `try` statement. This minimizes: "Where did this exception get thrown from?" Obviously this is an innocent enough change since we don't expect the `return` to raise, but I believe these are semantically equivalent so I'll stick to my rule of thumb here.